SU2_CFD/include/solvers/CTurbSSTSolver.hpp

```diff
@@ -42,6 +42,25 @@ class CTurbSSTSolver final : public CTurbSolver {
   kine_Inf,              /*!< \brief Free-stream turbulent kinetic energy. */
   omega_Inf;             /*!< \brief Free-stream specific dissipation. */
 
+
+
+/*!
+   * \brief Compute nu tilde from the wall functions.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] conv_numerics - Description of the numerical method.
+   * \param[in] visc_numerics - Description of the numerical method.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] val_marker - Surface marker where the boundary condition is applied.
+   */
+  void SetNuTilde_WF(CGeometry *geometry,
+                     CSolver **solver_container,
+                     CNumerics *conv_numerics,
+                     CNumerics *visc_numerics,
+                     CConfig *config,
+                     unsigned short val_marker);
```

Comment:
I would name this method after the kw variables it manipulates instead, the argument list can also probably be reduced? e.g. the numerics are not used in its implementation.

Reply:
Since the method is not virtual it is not forced to follow the naming and signature of any other method of any solver.